save-demo/src/main/kotlin/com/saveourtool/save/demo/SaveDemo.kt

```diff
@@ -3,13 +3,17 @@ package com.saveourtool.save.demo
 import com.saveourtool.save.demo.config.ConfigProperties
 import org.springframework.boot.SpringApplication
 import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.autoconfigure.domain.EntityScan
 import org.springframework.boot.context.properties.EnableConfigurationProperties
 
 /**
  * An entrypoint for spring boot for save-demo
  */
 @EnableConfigurationProperties(ConfigProperties::class)
 @SpringBootApplication
+@EntityScan(
+    "com.saveourtool.save.demo.entity",
```

Comment:
do we really need it? `@SpringBootApplication` should scan `com.saveourtool.save.demo` automatically

Reply:
No, we do not, will remove it

Reply:
Done